### PR TITLE
Build text input field

### PR DESCRIPTION
**Pull request by sachasmart-weavik**
### Description

> Updates to Text input component. Includes focused, unfocused, and error states. 
### Checklist

- [x] `CHANGELOG` has been updated (_if appropriate_)
- [x] Environment variables updated (_if appropriate_)

### CODE REVIEW

File: packages/client3/components/design-system/input/input.tsx [hunk 1]
```diff
@@ -163,14 +150,8 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
...
           className={clsx(
-            isError
-              ? "text-clay-500"
-              : colorScheme === "light"
-              ? "text-sand-500"
-              : colorScheme === "dark"
-              ? "text-sand-300"
-              : null,
-            "mt-1 text-sm leading-none"
+            isError ? "" : null,
```
Comment (v1n33th) [inline packages/client3/components/design-system/input/input.tsx:153]: I believe this isn't necessary

File: packages/client3/components/design-system/input/input.tsx [hunk 2]
```diff
@@ -163,14 +150,8 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
...
+            "text-md mt-1 leading-none text-state-error"
```
Comment (v1n33th) [inline packages/client3/components/design-system/input/input.tsx:154]: Just to make sure is this style (`text-state-error`) applied for helpertext as well

File: packages/client3/components/design-system/input/input.tsx [hunk 3]
```diff
@@ -112,22 +106,15 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(function Input(
...
-              : colorScheme === "dark"
-              ? [
-                  "border bg-sky-900 focus:border-white",
-                  isError
-                    ? "border-clay-500 placeholder:text-clay-500"
-                    : "border-transparent placeholder:text-sand-300",
-                ]
-              : null,
+            "unfocused w-full rounded border",
+            // unfocused because the color schemes supply a border color as a focus style
```
Comment (v1n33th) [inline packages/client3/components/design-system/input/input.tsx:110]: Now that color scheme is removed from props, does this comment hold value?

**Review by v1n33th — COMMENTED** [review]
Awesome work..Just few questions.

File: packages/client3/components/design-system/input/input.tsx (hunk 2 above)
Comment (sachasmart-weavik) [reply packages/client3/components/design-system/input/input.tsx:154]: Yeap, confirmed that the - `text-state-error` is applied to the `HelperText`

File: packages/client3/components/design-system/input/input.tsx (hunk 3 above)
Comment (sachasmart-weavik) [reply packages/client3/components/design-system/input/input.tsx:110]: Fair enough. Taken out.

File: packages/client3/components/design-system/input/input.tsx (hunk 2 above)
Comment (v1n33th) [reply packages/client3/components/design-system/input/input.tsx]: Oops, my bad. I wasn't clear.  My query was whether the  `text-state-error` needs to be applied to `HelperText`?

Comment (sachasmart-weavik) [reply packages/client3/components/design-system/input/input.tsx]: Whoops my bad. Pushed the fix: 


>         <HelperText
>           className={clsx(
>             isError ? "text-state-error" : null,
>             "mt-1 text-sm leading-none"
>           )}
>         >
>           {_errorMessage ? _errorMessage : helperText}
>         </HelperText>

**Review by v1n33th — APPROVED** [review]

**Comment by luansievers** [comment]
@sachasmart-weavik can you share an image of the input?

**Comment by sachasmart-weavik** [comment]
Figma: https://www.figma.com/file/g7Edk2JCOMLpRAw9v8NJ4c/FA-Product-Wireframe---working-file?node-id=136%3A6264


Screen Shot 2022-09-01 at 3 05 52 PM (2)

ezgif com-gif-maker

**Pull request merged**